scripts/lib/touchstone-plan-records.sh

```
@@ -46,9 +91,21 @@ plan_name_exists() {
 }
 
 append_plan_record() {
-  local output="$1" failure="$2" format="$3"
+  local output="$1" failure="$2" format="$3" record
   shift 3
-  printf "$format" "$@" >>"$output" || operational_failure "$failure"
+  if [ "$PLAN_IN_MEMORY" = false ]; then
+    printf "$format" "$@" >>"$output" || operational_failure "$failure"
```

Shellcheck (0.11.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)